server/src/main/java/com/cloud/network/NetworkModelImpl.java

```diff
@@ -1659,7 +1659,7 @@ public void checkCapabilityForProvider(Set<Provider> providers, Service service,
     }
 
     @Override
-    public void checkNetworkPermissions(Account owner, Network network) {
+    public void checkNetworkPermissions(Account caller, Network network) {
```

Comment:
could we explore if a unit test is written

Reply:
added some unit tests